pytorch/inference/docker/1.5.1/py38/Dockerfile.eia

```diff
@@ -93,6 +99,7 @@ RUN conda install -y \
  && pip install numpy==1.22.4 \
  && pip install pillow==9.2.0 \
  && pip install urllib3==1.26.11 \
+ && pip install wheel==0.38.1 \
```

Comment:
Is this line required? I this can be removed since `Wheel` is being installed as a dependency of some other package. We can remove this line altogether and the latest version of wheel should install automatically.